project/src/main/java/com/google/sps/data/Places.java

```diff
@@ -29,11 +29,11 @@
 
   /**
    * Sorts the given list of Places randomly.
-   * @param places the list we want to sort.
+   * @param immutableList the list we want to sort.
```

Comment:
I don't understand the naming, the fact that it's an immutable list is reflected in the type, and I don't think the new name expresses the essence of the variable... What do you think?

Reply:
Oh I didn't notice the name change, maybe I accidently changed it but I don't remeber.. good that you noticed!
The only thing I meant to change is the ImmutableList import because it gave us errors that we used different imports for this in every file.